PendingReleaseNotes.md

```diff
@@ -2,10 +2,11 @@
 
 ## Breaking Changes
 
-*  The mds liveness and startup probes are now configured by the filesystem CR instead of the cluster CR. To apply the mds probes, they need to be specified in the filesystem CR. See the [filesystem CR doc](Documentation/ceph-filesystem-crd.md#metadata-server-settings) for more details. 
-Pr: https://github.com/rook/rook/pull/9550
+*  The mds liveness and startup probes are now configured by the filesystem CR instead of the cluster CR. To apply the mds probes, they need to be specified in the filesystem CR. See the [filesystem CR doc](Documentation/ceph-filesystem-crd.md#metadata-server-settings) for more details. See #9550
 
 ## Features
 
-*  Add support for custom ceph.conf for csi pods.
-Pr: https://github.com/rook/rook/pull/9567
+* The default number of mgr daemons is increased to 2 from 1, resulting in a standby mgr daemon.
+  If the active mgr goes down, Ceph will update the passive mgr to be active, and rook will update
+  all the services with the label app=rook-ceph-mgr to direct traffic to the new active mgr.
```

Comment:
Is there an internal default value we should change as well? Otherwise, maybe this would be better worded as "change the production example manifests to use 2 mgr daemons instead of 1" since we haven't really changed builtin defaults.

Here maybe:
https://github.com/BlaineEXE/rook/blob/2d8f9b67c63d67acae9fca98e3e1cafd0f716e3e/pkg/operator/ceph/cluster/mgr/mgr.go#L91-L98

Reply:
I'm thinking we just change the default for **new** clusters. Upgraded clusters can keep the single mgr until the setting is changed.